.circleci/images/test-image-arm64/config_linux5.2.0_arm64.txt

```diff
@@ -0,0 +1,7454 @@
+#
```

Comment:
This may be controversial, but I'd like it if this was like the amd64 config file: https://github.com/u-root/u-root/blob/master/.circleci/images/test-image-amd64/config_linux4.17_x86_64.txt

It's hard to tell which configs here are for the tests. It'd be nice if this config was a "delta" from a defconfig: you just place the configs that *aren't* turned on by default in here, and use make olddefconfig in the dockerfile to generate the full config.

That's how the amd64 config works, and imho it makes it readable.

(It doesn't make it a super small kernel, but I think for the VM tests that doesn't matter anyway.)

Reply:
thanks chris, def agree with you on the unwieldy config file. I put in a modified version of the config file that should be a little more readable. I wasn't sure the minimal subset of features were required so I've included all of them for now.

Reply:
yeah when I shortened the other config, it required some trial and error.

that config is probably not bad to start with, as long as you change the arch to amd64. and then you could see what still breaks?

you'll need the same features where they're shared between amd64 and arm64 anyway, and then the difference will be the arm64-specific stuff (of which i'm not sure how much there is... @rjoleary will know?)

Reply:
Everything else looks good!

I'm not too sure about the additional configs for arm64. The base configs you will need are likely similar to the list for arm: https://github.com/u-root/u-root/blob/master/configs/arm_config.txt

Reply:
Got it to be pretty much the same size as the arm configs. Only had to add 2 extra config options to make it work with the integration tests.